lib/log.js

```diff
@@ -32,25 +32,30 @@ module.exports = function log(tree, _options) {
       return readTree(tree).then(function(dir) {
         var label = chalk.bold.cyan;
 
-        if (options.output === 'tree') {
-          var printTree = treeOutput(dir);
+        if (process.env.DEBUG === 'true') {
```

Comment:
this should actually just be `if (process.env.DEBUG && new RegExp(process.env.DEBUG).test(label))` as DEBUG can be a matching string

see: https://github.com/visionmedia/debug#wildcards